src/appProvider.js

```diff
@@ -42,7 +42,7 @@ export default class AppProvider extends Component {
               <ConnectedRouter history={history}>
                 <Route
                   path="/"
-                  render={(props) => (<App match={props.match} langHandler={this.toggleLanguage} />)}
+                  render={(props) => (<App match={props.match} langHandler={this.toggleLanguage} locale={this.state.locale} />)}
```

Comment:
You don't need to do this. You can get the locale in `App/index.js` by using the `@injectIntl` and then it will be in `this.props.intl.locale`. That's the point of having the `<IntlProvider locale={this.state.locale}>`

Reply:
good to know thanks! fixed